tilt-provider.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "digitalocean",
+  "config": {
+    "image": "gcr.io/k8s-staging-cluster-api-digitalocean/cluster-api-digitalocean-controller",
```

for now, this does not exist, but i think we should start pushing the images as soon we have the e2e or don't need? then I will check what we need to do to have the bucket

---

```suggestion
    "image": "gcr.io/k8s-staging-cluster-api-do/cluster-api-digitalocean-controller",
```

The GCR project is called `k8s-staging-cluster-api-do`.